src/otx/cli/cli.py

```diff
@@ -331,11 +332,23 @@ def instantiate_classes(self, instantiate_engine: bool = True) -> None:
             # For num_classes update, Model and Metric are instantiated separately.
             model_config = self.config[self.subcommand].pop("model")
 
+            # if adaptive_input_size will be executed and the model has input_size_multiplier, pass it to OTXDataModule
+            if self.config[self.subcommand].data.adaptive_input_size != "none":
+                model_cls = get_model_cls_from_config(model_config)
+                self.config[self.subcommand].data.input_size_multiplier = model_cls.input_size_multiplier
+
             # Instantiate the things that don't need to special handling
             self.config_init = self.parser.instantiate_classes(self.config)
             self.workspace = self.get_config_value(self.config_init, "workspace")
             self.datamodule = self.get_config_value(self.config_init, "data")
 
+            # pass OTXDataModule input size to the model
+            if (input_size := self.datamodule.input_size) is not None and "input_size" in model_config["init_args"]:
+                # TODO(eunwoosh): After configurable input size is applied to anomaly, remove input_size check
```

Comment:
I think it would be better to just let Anomaly take `input_size` as well. Are we going to have trouble with this?

Reply:
Anomaly has complex hierarchical structure, which makes it hard to update input_size feature on anomaly task. And it's being refactored So, I excluded anomaly tasks from this PR. I think it's better to update it after refactoring is done.